task3/src/routers/controllers/user.controller.ts

```diff
@@ -0,0 +1,96 @@
+import { Request, Response } from 'express';
```

Comment:
You have a duplicate of this file under services folder. However, current implementation does not map to 3-layer architecture. You should not describe logic related to db requests here. Use service for that. In other words, leave methods for requests to db in services and use them here.

Reply:
Rewite